batch.go

```diff
@@ -973,7 +982,10 @@ func (batch *Batch) addendaFieldInclusionReturn(entry *EntryDetail) error {
 	if entry.Addenda98 != nil {
 		return batch.Error("Addenda98", ErrBatchAddendaCategory, entry.Category)
 	}
-	if entry.Addenda99 == nil {
+	if entry.Addenda04 == nil && entry.Addenda99 == nil {
+		if entry.Addenda04 == nil {
+			return batch.Error("Addenda04", ErrFieldInclusion)
+		}
```

Comment:
am I right that line 989 (below) will never be run because `entry.Addenda04 == ` is always true if we are inside?

Reply:
Yea.. That's a good point.